src/Polly.RateLimiting/RateLimiterRejectedException.cs

```diff
@@ -84,10 +131,16 @@ public RateLimiterRejectedException(string message, TimeSpan retryAfter, Excepti
     private RateLimiterRejectedException(SerializationInfo info, StreamingContext context)
         : base(info, context)
     {
-        var value = info.GetDouble("RetryAfter");
-        if (value >= 0.0)
+        var retryAfter = info.GetDouble(nameof(RetryAfter));
+        if (retryAfter >= 0.0)
```

Comment:
Seeing this again, I wonder if this should be `default` or `0d`?

Reply:
Ignore this - I thought we'd recently added this but it hadn't been released yet, but actually that was `BrokenCircuitException`, so changing this would break compatibility.